### ISSUE TITLE
Differential Loading causes Subresource Integrity hashes to be incorrect

### ISSUE THREAD

**Issue by trickeyone** [angular/angular-cli#19799]
# 🐞 Bug report

### Command (mark with an `x`)

- [ ] new
- [x] build
- [ ] serve
- [ ] test
- [ ] e2e
- [ ] generate
- [ ] add
- [ ] update
- [ ] lint
- [ ] extract-i18n
- [ ] run
- [ ] config
- [ ] help
- [ ] version
- [ ] doc

### Is this a regression?

Unsure. SRI was not used before.

### Description

When setting the `target` in my tsconfig.json to "es2015", the ES5 differential loading generation is done. This is causing the SRI hashes that had been created during the build to become incorrect. It also appears to duplicate emitted assets to webpack. I have seen older issues that mention this, especially in regards to SRI and differential loading. The only "fix" determined in those issues was to set the target back to "es5". I can confirm that this does prevent the duplicate emissions and that the SRI hashes remain intact and correct.

```
$ ng build --prod

Generating ES5 bundles for differential loading...
ES5 bundle generation complete.

chunk {9} 9.29ab9ec671705879bb89.js () 1.18 kB  [rendered]
chunk {9} 9.29ab9ec671705879bb89.js () 1.18 kB  [rendered]
chunk {0} runtime.e6a9815e4f5e3997200e.js (runtime) 2.36 kB [entry] [rendered]
chunk {0} runtime.e6a9815e4f5e3997200e.js (runtime) 2.36 kB [entry] [rendered]
chunk {11} 11.a93b838b879521f1e9ca.js () 1.56 kB  [rendered]
chunk {11} 11.a93b838b879521f1e9ca.js () 1.55 kB  [rendered]
chunk {10} 10.6264e0a5f50992407853.js () 9.18 kB  [rendered]
chunk {10} 10.6264e0a5f50992407853.js () 10.3 kB  [rendered]
chunk {6} polyfills.26499ce4de4576627744.js (polyfills) 38.9 kB [initial] [rendered]
chunk {6} polyfills.26499ce4de4576627744.js (polyfills) 42.1 kB [initial] [rendered]
chunk {7} polyfills-es5.17a437d28be0f3f3152a.js (polyfills-es5) 132 kB [initial] [rendered]
chunk {3} login.2a28ffc20e1e6f62bf81.js (login) 733 kB [initial] [rendered]
chunk {3} login.2a28ffc20e1e6f62bf81.js (login) 733 kB [initial] [rendered]
chunk {5} main.b32632312744b0f8356d.js (main) 3.81 MB [initial] [rendered]
chunk {5} main.b32632312744b0f8356d.js (main) 4.02 MB [initial] [rendered]
chunk {1} all.6915c1bc.css (all) 613 kB [initial] [rendered]
chunk {document-register-element} document-register-element.dc357b0f6dff293d5fa6.js (document-register-element) 13.5 kB [entry] [rendered]
chunk {google-analytics} google-analytics.ebbc4b2aea3eabd7f70b.js (google-analytics) 371 bytes [entry] [rendered]
Date: 2021-01-14T22:10:42.531Z - Hash: 4640c90825df2401ff0c - Time: 200753ms
```

## 🔬 Minimal Reproduction

I can't be 100% about this, but I believe simply enabling es2015 and SRI would reproduce the issue.

## 🔥 Exception or Error
No errors occur

## 🌍 Your Environment
<pre><code>

     _                      _                 ____ _     ___
    / \   _ __   __ _ _   _| | __ _ _ __     / ___| |   |_ _|
   / △ \ | '_ \ / _` | | | | |/ _` | '__|   | |   | |    | |
  / ___ \| | | | (_| | |_| | | (_| | |      | |___| |___ | |
 /_/   \_\_| |_|\__, |\__,_|_|\__,_|_|       \____|_____|___|
                |___/
    

Angular CLI: 10.2.1
Node: 12.20.1
OS: darwin x64

Angular: 10.2.4
... animations, common, compiler, compiler-cli, core, elements
... forms, localize, platform-browser, platform-browser-dynamic
Ivy Workspace: Yes

Package                            Version
------------------------------------------------------------
@angular-devkit/architect          0.1002.1
@angular-devkit/build-angular      0.1000.8
@angular-devkit/core               10.2.1
@angular-devkit/schematics         10.2.1
@angular/cdk                       10.2.7
@angular/cli                       10.2.1
@angular/material                  10.2.7
@angular/material-moment-adapter   10.2.7
@ngtools/webpack                   10.2.1
@schematics/angular                10.2.1
@schematics/update                 0.1002.1
rxjs                               6.6.3
typescript                         3.9.7
webpack                            4.46.0

</code></pre>

**Anything else relevant?**

**Comment by alan-agius4** [comment]
Thanks for reporting this issue. Luckily, it has already been fixed in one of the recent releases. Please update to the most recent version to resolve the problem.

If the problem persists in your application after upgrading, please open a new issue, provide a simple repository reproducing the problem, and describe the difference between the expected and current behavior. You can use `ng new repro-app` to create a new project where you reproduce the problem.

**Comment by trickeyone** [comment]
Hi @alan-agius4. I'm on the latest version for v10. Is the fix only available in v11?

**Comment by alan-agius4** [comment]
Just checked and it’s also available in the v10.2.

kindly run the following command.

```
ng update @angular/cli@v10-lts
